src/main/kotlin/no/nav/familie/ef/sak/health/InfotrygdFeedHealth.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.ef.sak.health
+
+import no.nav.familie.ef.sak.integration.InfotrygdFeedClient
+import no.nav.familie.http.health.AbstractHealthIndicator
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+
+@Component
+@Profile("!local")
```

Comment:
local = mock?

Reply:
Lokal brukes for lokal kjøring av ef-sak, er likt i FamilieIntegrasjonHealth :)